content/integrations/amazon_web_services.fr.md

```diff
@@ -2,7 +2,8 @@
 aliases:
   - /fr/integrations/aws/
   - /fr/logs/aws
-  - /fr/integrations/faq/revoking-aws-keys-and-enabling-role-delegation-for-the-datadog-aws-integration/
+  - >-
+    /fr/integrations/faq/revoking-aws-keys-and-enabling-role-delegation-for-the-datadog-aws-integration/
```

Comment:
@l0k0ms This appears to be valid yaml and is something called `block chomping indicator` 

https://stackoverflow.com/a/3790497/1113788
https://yaml.org/spec/1.2/spec.html#id2794534

As long as it didn't cause hugo to fail it should be fine for now?

Reply:
yup, didn't know that thanks for the info 🙇

Reply:
I found a setting that should reduce the chances of this happening by increasing line length. I'm releasing it now

Reply:
🙇